README.md

```diff
@@ -32,32 +29,32 @@ cuVS contains state-of-the-art implementations of several algorithms for running
 
 ## Installing cuVS
 
-cuVS comes with pre-built packages that can be installed through [conda](https://conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-python). Different packages are available for the different languages supported by cuVS:
+cuVS comes with pre-built packages that can be installed through [conda](https://conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-python) and [pip](https://pip.pypa.io/en/stable/). Different packages are available for the different languages supported by cuVS:
 
-| Python | C/C++                       |
-|--------|-----------------------------|
-| `cuvs` | `libcuvs`, `libcuvs-static` |
+| Python | C/C++     |
+|--------|-----------|
+| `cuvs` | `libcuvs` |
 
 ### Stable release
 
-It is recommended to use [mamba](https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html) to install the desired packages. The following command will install the Python package. You can substitute `cuvs` for any of the packages in the table above:
+It is recommended to use [mamba](https://conda.github.io/conda-libmamba-solver/user-guide/) to install the desired packages. The following command will install the Python package. You can substitute `cuvs` for any of the packages in the table above:
 
 ```bash
-mamba install -c conda-forge -c nvidia -c rapidsai cuvs
+conda install -c conda-forge -c nvidia -c rapidsai cuvs
```

Comment:
should we keep the command as `mamba` since thats our recommendation to use in the above paragraph?
```suggestion
mamba install -c conda-forge -c nvidia -c rapidsai cuvs
```

Reply:
Actually, we got dinged on this in our VDR because `mamba` apparenlty only works up to Python 3.11, while conda uses the `libmamba` backend direclty. That's one reason I linked to the instructions for enabling the libmamba backend within conda and changed this. Apparently it was too confusing installing mamba for the VDR team whereas conda at least works out of the box.

Reply:
Let me know if there's a better way to word the suggestion to use mamba, or a better set of instructions we should be linking to.